README.md

```diff
@@ -17,7 +17,7 @@ This extension assumes you have the following binaries on your `PATH`:
    * `draft` (optional)
    * `minikube` (optional)
 
-For `kubectl`, `helm` and `draft` the binaries need not be on the system PATH, provided you tell the extension their locations using the appropriate `vs-kubernetes -> vs-kubernetes.${tool}-path` configuration setting.  See [Extension Settings](#extension-settings) below.
+For `kubectl`, `helm` and `draft` the binaries need not to be on the system PATH. You tell to the extension their locations using the appropriate `vs-kubernetes -> vs-kubernetes.${tool}-path` configuration setting.  See [Extension Settings](#extension-settings) below.
```

Comment:
In the second sentence, please remove 'to' - should be 'You tell the extension...'  Also, do you think something like "In this case, you tell the extension..." would be clearer still?  Not sure and would value your feedback.

Reply:
i suggest "You can configure the extension by specifying the locations"

Reply:
Thank you for your kindness